flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/ExecutionContext.java

```diff
@@ -242,6 +249,36 @@ public EnvironmentInstance createEnvironmentInstance() {
 
 	// --------------------------------------------------------------------------------------------
 
+	private static Configuration createExecutionConfig(
+			CommandLine commandLine,
+			Options commandLineOptions,
+			List<CustomCommandLine> availableCommandLines) throws FlinkException {
+
```

Comment:
This `options` is only used for logging. Do we need it or we can simply print the final `executionConfig `?